package.nls.json

```diff
@@ -108,7 +108,8 @@
     "AWS.command.refreshAwsExplorer": "Refresh Explorer",
     "AWS.command.refreshCdkExplorer": "Refresh CDK Explorer",
     "AWS.command.cdk.help": "View CDK Documentation",
-    "AWS.command.ec2.connectToInstance": "Connect to EC2 Instance...",
+    "AWS.command.ec2.openTerminal": "Open terminal to EC2 instance...",
+    "AWS.command.ec2.openRemoteConnection": "Connect to EC2 instance in New Window...",
```

Comment:
This is for connecting vscode-remote, right? Then we don't need to mention "new window" because we don't have control over that. But we should mention "IDE" perhaps.

    Connect IDE to EC2 instance...

Future note: We eventually will want to think about being consistent across all of our "remote" features. For CodeCatalyst this feature is called `Open Dev Environment`, but maybe it should be called `Connect IDE to Dev Environment`.